collections/Laud_Misc/MS_Laud_Misc_636.xml

```diff
@@ -37,43 +59,195 @@
                      <idno type="SCN">1003</idno>
                   </altIdentifier>
                </msIdentifier>
+               <head>The Peterborough Chronicle (Anglo-Saxon Chronicle, MS. E); Peterborough, c. 1121-31, c. 1155, and s. xiii/xiv</head>
                <msContents>
-                  <msItem class="#historica" n="1" xml:id="MS_Laud_Misc_636-item1">
-                     <title key="work_10926" type="desc">Chronicle of <placeName key="place_7002445">England</placeName> (Anglo-Saxon Chronicle) to 1154 ('The Peterborough Chronicle')</title>
+                  <summary>The so-called Anglo-Saxon chronicle was originally compiled in King Alfred's reign and perhaps with his encouragement, and was continued in several versions after his death (Irvine 2015). This manuscript, copied at Peterborough Abbey, is the only version to continue beyond 1079, and is an almost unique witness to historical writing in Old English in the early twelfth century. The later annals are particularly celebrated for the supposedly eye-witness description of William the Conqueror, and for their evidence of linguistic change from Old English to Middle English.</summary>
```

Comment:
I would personally advocate cutting this, since the catalogue rarely includes such discussions.